src/styles/annotator/components/AdderToolbar.scss

```diff
@@ -51,18 +43,22 @@ $adder-transition-duration: 80ms;
 }
 
 .AdderToolbar__arrow {
-  @include molecules.menu-arrow;
+  position: absolute;
+  z-index: 2;
```

Comment:
I have tested this z-index in this page: http://localhost:3000/document/z-index. It works properly because it is in a shadow-dom and the wrapper container set the correct z-index.